docs/building/forms.md

```diff
@@ -21,36 +21,60 @@ Form
+        "float"     - the user could enter a floating number
+        "date"      - the user should enter a date
+        "datetime"  - the user should enter a date and time
+        "form-menu" - the user should choose an option from the menu
+
+    name                       - (string) the name of this FormItem, used in form serialization
+
+    body                       - (array of objects) MenuItemFormItem objects - see below - required only for type=form-menu
+    chunking_footer            - (string) shown in the footer of the sms chunks
+    confirmation_label         - (string) shown in the confirmation menu
+    description                - (string) the description of this FormItem
+    footer                     - (string) if provided will overwrite the Form.footer
+    header                     - (string) if provided will overwrite the Form.header
+    max_length                 - (int) validates the user input - for type=string
+    max_length_error           - (string) message to be shown on max_length error
+    max_value                  - (int) validates the user input - for type=int|float
+    max_value_error            - (string) message to be shown on max_value error
+    meta                       - (object) MenuFormItemMeta object - see below - applies only for type=form-menu
+    method                     - (string) http method, how the callback url should be triggered
+    min_length                 - (int) validates the user input - for type=string
+    min_length_error           - (string) message to be shown on min_length error
+    min_value                  - (int) validates the user input - for type=int|float
+    min_value_error            - (string) message to be shown on min_value error
+    required                   - (bool) can be skipped if set to false
+    status_exclude             - (bool) if true this step will be excluded from the form completion status
+    status_prepend             - (bool) if true this step will be prepended to the body pre of the response - appended otherwise
+    url                        - (string) callback url triggered right after the choice has been set for this form item
+    validate_type_error        - (string) an error message to be shown on basic type validation
+    validate_type_error_footer - (string) shown in the error message footer
+    validate_url               - (string) the callback url path (GET) triggered to validate user input with query string ?name=user_input - url must return json content {'valid': True/False, 'error': 'Some validation error message'}
+    value                      - (string) required for type=hidden
+    
+
+MenuItemFormItem
+    type - (string) enum: "option", "content"
+        "option"  - needs a value
+        "content" - does not need a value - shows content as indicated in description
+
+    value       - (string) the value of this FormItemMenuItem, used in form serialization
```

Comment:
You renamed this to `MenuItemFormItem`. See also the line below.

Reply:
Well spotted! Renamed here https://github.com/ONEmCommunications/documents/pull/12/commits/849d5a0b14049a9d6e2e50ba65c31d95c97b2fae